scripts/ansible/README.md

```diff
@@ -127,76 +127,80 @@ Target operating systems supported by these roles & playbooks are
+changed: [localhost] => (item=avalanchego-linux-arm64-v1.5.0.tar.gz)
+changed: [localhost] => (item=avalanchego-linux-arm64-v1.5.0.tar.gz.sig)
 
-TASK [avalanche_base : Install Avalanche dependencies] ********************************
-ok: [localhost]
+TASK [avalanche_download : Create temp gnupghome] ******************************
+changed: [localhost]
 
-TASK [avalanche_build : Update git clone] *********************************************
+TASK [avalanche_download : Import keys] ****************************************
 changed: [localhost]
 
-TASK [avalanche_build : Build project] ************************************************
+TASK [avalanche_download : Verify signature] ***********************************
+ok: [localhost]
+
+TASK [avalanche_download : Cleanup temp gnupghome] *****************************
 changed: [localhost]
 
-TASK [avalanche_user : Create Avalanche daemon group] *********************************
+TASK [avalanche_download : Unpack release] *************************************
+changed: [localhost] => (item=avalanchego-linux-arm64-v1.5.0.tar.gz)
+
+TASK [avalanche_user : Create Avalanche daemon group] **************************
 changed: [localhost]
 
-TASK [avalanche_user : Create Avalanche daemon user] **********************************
-[WARNING]: The value False (type bool) in a string field was converted to 'False'
-(type string). If this does not look like what you expect, quote the entire value to
-ensure it does not change.
+TASK [avalanche_user : Create Avalanche daemon user] ***************************
 changed: [localhost]
 
-TASK [avalanche_install : Create shared directories] **********************************
-ok: [localhost] => (item={'path': '/var/local/lib'})
+TASK [avalanche_install : Create shared directories] ***************************
+changed: [localhost] => (item={'path': '/usr/local/bin'})
+changed: [localhost] => (item={'path': '/var/local/lib'})
 changed: [localhost] => (item={'path': '/var/local/log'})
 
-TASK [avalanche_install : Create Avalanche directories] *******************************
-ok: [localhost] => (item=/var/local/lib/avalanchego)
+TASK [avalanche_install : Create Avalanche directories] ************************
+changed: [localhost] => (item=/var/local/lib/avalanchego)
 changed: [localhost] => (item=/var/local/lib/avalanchego/db)
 changed: [localhost] => (item=/var/local/lib/avalanchego/staking)
 changed: [localhost] => (item=/var/local/log/avalanchego)
 changed: [localhost] => (item=/usr/local/lib/avalanchego)
 
-TASK [avalanche_install : Install Avalanche binary] ***********************************
+TASK [avalanche_install : Install Avalanche binary] ****************************
 changed: [localhost]
 
-TASK [avalanche_install : Install Avalanche plugins] **********************************
-changed: [localhost] => (item={'path': '~auser/go/src/github.com/ava-labs/avalanchego/build/plugins/evm'})
+TASK [avalanche_install : Remove outdataed support files] **********************
```

Comment:
super minor nit (that isn't really related to this PR) but I just noticed that this seems to be a typo.

outdataed -> outdated

Reply:
ty, I'll do a sweep for others as well